UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/ViewComponents/BannerNotificationViewComponent.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.ViewComponents
+{
+    [ViewComponent(Name = "BannerNotification")]
+    public class BannerNotificationViewComponent : BaseViewComponent<BannerNotificationViewComponent>
+    {
+        private readonly IMSIBannerNotificationService _mSIBannerNotificationService;
+        private readonly ILogger<BannerNotificationViewComponent> _logger;
+        private readonly IUserService _userService;
+
+        public BannerNotificationViewComponent(IHttpContextAccessor contextAccessor, ILogger<BannerNotificationViewComponent> logger, IMSIBannerNotificationService mSIBannerNotificationService, IUserService userService) : base(contextAccessor, logger)
+        {
+            _logger = logger;
+            _mSIBannerNotificationService = mSIBannerNotificationService;
+            _userService = userService;
+        }
+
+        public async Task<IViewComponentResult> InvokeAsync()
+        {
+            try
+            {
+                _logger.LogError(EventIds.BannerNotificationRequestStarted.ToEventId(), "Maritime safety information request to get banner notification message started for User:{SignInName} and IsDistributor:{IsDistributorUser} with _X-Correlation-ID:{CorrelationId}", _userService.SignInName ?? "Public", _userService.IsDistributorUser, GetCurrentCorrelationId());
```

Comment:
This log's type should be information instead of error.
```suggestion
                _logger.LogInformation(EventIds.BannerNotificationRequestStarted.ToEventId(), "Maritime safety information request to get banner notification message started for User:{SignInName} and IsDistributor:{IsDistributorUser} with _X-Correlation-ID:{CorrelationId}", _userService.SignInName ?? "Public", _userService.IsDistributorUser, GetCurrentCorrelationId());
```

Reply:
done required changes.